DxDispatch/src/model/OnnxParsers.cpp

```diff
@@ -88,9 +89,14 @@ Model OnnxParsers::ParseModel(
     sessionOptions.DisableMemPattern();
     sessionOptions.SetGraphOptimizationLevel(GraphOptimizationLevel::ORT_ENABLE_EXTENDED);
 
-    for (auto& freeDimOverride : freeDimOverrides)
+    for (auto& freeDimOverride : freeDimNameOverrides)
```

Comment:
>for (auto& freeDimOverride : freeDimNameOverrides) [](http://example.com/codeflow?start=4&length=50)

I suppose it would be overkill to put this repeated snippet into a separate function. Perhaps it gets more complicated later though.

Reply:
The duplicated logic in OnnxParsers.cpp and OnnxDispatchable.cpp bothers me, but not enough to refactor right now. It would be preferable to reflect the input/output dimensions/types from the protobuf directly instead of instantiated an entire ORT session to read this info. :(